### PR TITLE
Added MERY.jp

### PR DESCRIPTION
**Pull request by luixxiul**

### CODE REVIEW

File: src/chrome/content/rules/MERY.jp.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```
Comment (J0WI) [inline src/chrome/content/rules/MERY.jp.xml]: Why are they problematic?
They all WFM.

Please also add www. (redirects to plain ^)

Comment (luixxiul) [inline src/chrome/content/rules/MERY.jp.xml]: Don't we need to remove traffics through Akamai? https://github.com/EFForg/https-everywhere/pull/6100#issuecomment-238074839
I'm confused here :-/

Comment (luixxiul) [inline src/chrome/content/rules/MERY.jp.xml]: Or should I just rename it nonfunctional?

Comment (J0WI) [inline src/chrome/content/rules/MERY.jp.xml]: I think it's just a problem when you redirect something like `images.foo.bar` to `eu1337.cdn.com/foo-images`, but as long as content live on their own domain it souldn't be a problem.

Comment (luixxiul) [inline src/chrome/content/rules/MERY.jp.xml]: I see, later I will review my other PRs which include akamai traffics.

Comment (fuglede) [inline src/chrome/content/rules/MERY.jp.xml]: Ah, indeed, and even then, redirecting to another domain can be completely fine; it's just that it turned out to break a lot of rulesets in the case of Akamai at one point in time. Apologies for any confusion.

Comment (luixxiul) [inline src/chrome/content/rules/MERY.jp.xml]: I see, no problem. Maybe we'll have to take some time to enable the domains which have been excluded due to the case ;-)

**Comment by luixxiul** [comment]
@J0WI OK, I updated the ruleset.

**Comment by J0WI** [comment]
Thanks!

**Comment by J0WI** [comment]
Could you also add/document www.?

**Comment by J0WI** [comment]
Oh and could you also include some test urls for the CDN hosts? Then we will be more likely aware if they change something that breaks our rule.

**Comment by luixxiul** [comment]
@J0WI OK, done.

**Pull request merged**